client/order/index.js

```diff
@@ -129,102 +134,172 @@ jQuery( function ( $ ) {
 const DisputeNotice = ( { chargeId } ) => {
 	const { data: charge } = useCharge( chargeId );
 
-	if (
-		! charge?.dispute ||
-		! charge?.dispute?.evidence_details?.due_by ||
-		// Only show the notice if the dispute is awaiting a response.
-		! isAwaitingResponse( charge.dispute.status )
-	) {
+	if ( ! charge?.dispute ) {
 		return null;
 	}
 
 	const { dispute } = charge;
 
-	const now = moment();
-	const dueBy = moment.unix( dispute.evidence_details?.due_by );
-	const countdownDays = Math.floor( dueBy.diff( now, 'days', true ) );
+	let urgency = 'warning';
+	let actions;
 
-	// If the dispute is due in the past, we don't want to show the notice.
-	if ( now.isAfter( dueBy ) ) {
-		return;
-	}
+	// Refunds are only allowed if the dispute is an inquiry or if it's won.
+	const isRefundable =
+		isInquiry( dispute ) || [ 'won' ].includes( dispute.status );
+	const shouldDisableRefund = ! isRefundable;
+	let disableRefund = false;
 
-	const titleStrings = {
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		dispute_default: __(
-			// eslint-disable-next-line max-len
-			'This order has been disputed in the amount of %1$s. The customer provided the following reason: %2$s. Please respond to this dispute before %3$s.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		inquiry_default: __(
-			// eslint-disable-next-line max-len
-			'The card network involved in this order has opened an inquiry into the transaction with the following reason: %2$s. Please respond to this inquiry before %3$s, just like you would for a formal dispute.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		dispute_urgent: __(
-			'Please resolve the dispute on this order for %1$s labeled "%2$s" by %3$s.',
-			'woocommerce-payments'
-		),
-		// Translators: %1$s is the formatted dispute amount, %2$s is the dispute reason, %3$s is the due date.
-		inquiry_urgent: __(
-			'Please resolve the inquiry on this order for %1$s labeled "%2$s" by %3$s.',
-			'woocommerce-payments'
-		),
-	};
-	const amountFormatted = formatExplicitCurrency(
-		dispute.amount,
-		dispute.currency
-	);
+	let refundDisabledNotice = '';
+	if ( shouldDisableRefund ) {
+		const refundButton = document.querySelector( 'button.refund-items' );
+		if ( refundButton ) {
```

Comment:
The refund button isn't rendered for `lost` disputes, so the expected message is not rendered due to this `if`` statement.